torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1289,7 +1314,11 @@ RegisterOperators reg2({
       Operator(                                                             \
           "aten::_set_item(" decl_type "[](a!) l, int idx, " decl_type      \
           " el) -> " decl_type "[](a!)",                                    \
-          listSetItem<Shared<c_type>, c_type::ElemType>)
+          listSetItem<Shared<c_type>, c_type::ElemType>),                   \
+      Operator(                                                             \
+          "aten::pop(" decl_type "[](a!) self, int idx=-1)                  \
```

Comment:
Python allows `pop`ing with negative indices (e.g.

```python
a = [1, 2, 3]
a.pop(-2)  # return 2
```
) so using -1 as a flag won't allow that. We could instead have 2 schemas for `aten::pop`, one where its just `aten::pop(the_list)` and the other with `aten::pop(the_list, index)`

Reply:
I think this works - pop() with no argument has the same semantics as pop(-1)